languages/ocaml.json

```diff
@@ -26,5 +26,35 @@
     "decreaseIndentPattern": "^\\s*(end|done|with|in|else)\\b|^\\s*;;",
     "increaseIndentPattern": "\\b(begin|object)\\s*$|\\blet [a-zA-Z0-9_-]+( [^ ]+)* =\\s*$|method!?[ \\t]+.*=[ \\t]*$|->[ \\t]*$|\\b(for|while)(_lwt)?[ \\t]+.*[ \\t]+do[ \\t]*$|(\\btry(_lwt)?$|\\bif\\s+.*\\sthen$|\\belse|[:=]\\s*sig|=\\s*struct)\\s*$",
     "indentNextLinePattern": "(?!\\bif.*then.*(else.*|(;|[ \\t]in)[ \\t]*$))\\bif|\\bthen[ \\t]*$|\\belse[ \\t]*$$"
-  }
+  },
+  "onEnterRules": [
+    {
+      "beforeText": "\\(\\*\\*",
+      "action": {
+        "indent": "none",
+        "appendText": "    "
+      }
+    },
+    {
+      "beforeText": "\\(\\*",
+      "action": {
+        "indent": "none",
+        "appendText": "   "
+      }
+    },
+    {
+      "beforeText": "\\*\\)",
+      "action": {
+        "indent": "outdent",
+        "removeText": 3
+      }
+    },
+    {
+      "beforeText": "\\*\\*\\)",
+      "action": {
+        "indent": "outdent",
+        "removeText": 4
+      }
+    }
```

Comment:
Documentation comments are typically still closed with `*)`, so there should only be one rule for the comment closing.

In other words, the `*)` has to outdent both 3 and 4 spaces.

Reply:
Seems to work fine for me inside a function with `"removeText": 2`:

![recording](https://user-images.githubusercontent.com/25037249/163453145-e5bebb3f-21c1-4355-9f00-83b35802009e.gif)

Reply:
Oh yes, it does work. Perfect !
Thank you !